helpers/class.Http.php

```diff
@@ -239,7 +239,7 @@ public static function returnFile($filename, $contenttype = true, $svgzSupport =
     {
         if (tao_helpers_File::securityCheck($filename, true)) {
             if (file_exists($filename)) {
-                $mimeType = tao_helpers_File::getMimeType($filename, true);
+                $mimeType = tao_helpers_File::getMimeType($filename);
                 if ($contenttype) {
                     header('Content-Type: ' . $mimeType);
```

Comment:
```suggestion
                if ($contenttype) {
                    header('Content-Type: ' . tao_helpers_File::getMimeType($filename));
```

No need to actually read the file's MIME type if it's wasn't requested.